akka-remote/src/test/scala/akka/remote/artery/RemoteDeployerSpec.scala

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery
+
+import akka.testkit._
+import akka.actor._
+import akka.routing._
+import com.typesafe.config._
+import akka.ConfigurationException
+import akka.remote.RemoteScope
+
+object RemoteDeployerSpec {
+  val deployerConf = ConfigFactory.parseString("""
+      akka.actor.provider = "akka.remote.RemoteActorRefProvider"
+      akka.actor.deployment {
+        /service2 {
+          router = round-robin-pool
+          nr-of-instances = 3
+          remote = "akka://sys@wallace:2552"
```

Comment:
I think that is on purpose, for testing "reject remote deployment when the source requires LocalScope"